### PR TITLE
Fixes YouTube placeholder width in news releases.

### PR DESCRIPTION
**Pull request by ryanlanier**
Also Fixes red box h4 border radius issue: h4 heading's 2px border radius shows inside of the container's 0px border radius.

### CODE REVIEW

File: docroot/sites/all/themes/custom/wsdot_classic/css/styles.css [hunk 1]
```diff
@@ -821,6 +821,7 @@ div.workbench-info-block {
 /* YouTube player */
 a.play-button {position:absolute;top:50%;left:50%;width:100px;height:100px;margin-top:-50px;margin-left:-50px;text-indent: 100%;white-space: nowrap;overflow: hidden;background:url(../images/play.png) no-repeat;background-size:100%;}
 .youtube-container {display:block;width:510px;max-width:98%;margin:2em 0;}
+.panels-flexible-region-52-center .youtube-container {width:450px;}
```
Comment (waynedyck) [inline docroot/sites/all/themes/custom/wsdot_classic/css/styles.css]: Where is the `.panels-flexible-region-52-center` class currently being referenced on the site? I'm not seeing it anywhere.

File: docroot/sites/all/themes/custom/wsdot_classic/templates/panels-pane--block--block-11.tpl.php [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
   <?php print render($title_prefix); ?>
   <?php if ($title): ?>
-    <h4 <?php print $title_attributes; ?> style="color:#fff; background: #710101;">
+    <h4 <?php print $title_attributes; ?> style="color:#fff; background: #710101;border-radius: 0;">
```
Comment (waynedyck) [inline docroot/sites/all/themes/custom/wsdot_classic/templates/panels-pane--block--block-11.tpl.php]: I'm not seeing what the `border-radius: 0` changes? Was it affecting a specific browser?

Comment (ryanlanier) [reply docroot/sites/all/themes/custom/wsdot_classic/templates/panels-pane--block--block-11.tpl.php]: The red box container div.contextual-links-region.panel-pane pane-block.pane-block-11 has the following style: border: 1px solid #710101. It also has a defauly border radius of 0px. The red box heading h4.pane-title has a border radius of 2px, background color: #710101. This causes the white background of the container to show at the upper left and right corners of the heading in all browsers. It's barely perceptible, but maybe worth fixing.

Comment (ryanlanier) [reply docroot/sites/all/themes/custom/wsdot_classic/templates/panels-pane--block--block-11.tpl.php]: The red box container div.contextual-links-region.panel-pane.pane-block.pane-block-11 has the following style: border: 1px solid #710101. It also has a default border radius of 0px. The red box heading h4.pane-title has a border radius of 2px, background color: #710101. This causes the white background of the container to show at the upper left and right corners of the heading in all browsers. It's barely perceptible, but maybe worth fixing.

Comment (waynedyck) [reply docroot/sites/all/themes/custom/wsdot_classic/templates/panels-pane--block--block-11.tpl.php]: Thanks. I see it now. Good catch.

**Comment by waynedyck** [comment]
Merged with f931e864ae4b0393d35ed7d570c759412d4cc7ec

**Pull request closed without merging**